MAUI/TimePicker/getting-started.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: Getting started with .NET MAUI Time Picker control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Time Picker (SfTimePicker) control and its basic features.
+platform: maui
+control: SfTimePicker
+documentation: ug
+---
+
+# Getting started with .NET MAUI Time Picker (SfTimePicker)
+This section explains how to add the Time Picker control. This section covers only the basic features needed to get started with Syncfusion Time Picker.
```

Comment:
This section explains how to add the Time Picker control. It covers only the basic features needed to get started with the Syncfusion Time Picker.

Reply:
Content modified.